src/gui/creds/webflowcredentialsdialog.cpp

```diff
@@ -151,6 +155,11 @@ void WebFlowCredentialsDialog::customizeStyle()
 
 void WebFlowCredentialsDialog::slotShowSettingsDialog()
 {
+    if (_raiseDelayTimer.isActive())
```

Comment:
It feels odd (and heavy) to use a QTimer connected to nothing like this. I'd use a QTime instead and measure time since last time we entered in that method. Also would make the delay more explicit since it'd be part of the if here.

That being said, it all feels a bit like adding a workaround on top. You seem to imply the client is raising that dialog too much? Why not solve that instead?